modulefiles/orion.intel/fv3

```diff
@@ -0,0 +1,57 @@
+#%Module######################################################################
+##
+##    NEMS FV3 Prerequisites: Orion/Intel
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running FV3 under NEMS on Orion/Intel"
+}
+
+module-whatis "loads NEMS FV3 prerequisites for Orion/Intel"
+
+##
+## load contrib environment
+## load noaatools (slurm utils (arbitrary.pl  layout.pl))
+##
+module load contrib noaatools
+
+##
+## load programming environment
+## this typically includes compiler, MPI and job scheduler
+##
+module load intel/2018
```

Comment:
Going back to 2018? For the UFS public release, we used intel/2020. I guess you need this to provide developers a stable alternative platform, as close as possible to hera, when the latter will be occupied with the retrospectives.